packages/katello/katello/katello-backup-rotate-tar.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+## For this script it's advisable to use a shell, such as Bash,
+## that supports a TAR_FD value greater than 9.
+
+# This script taken from the tar info doc
+
+name=`expr ${TAR_ARCHIVE} : '\(.*\)\..*'`
+volnum=`printf "%04d" ${TAR_VOLUME}`
```

Comment:
Is there a way we can avoid the zero-padding and the "part"? i.e. `pulp_data.2` instead of  `pulp_data.part0002`

Also is this still a tar file? Should it have a `.tar` extension?

Reply:
I think `pulp_data.2.tar` would be fine unless I'm missing out on any convention around this

Reply:
I'd prefer to have some leading zeros so that they sort nicely for an `ls`.

The files themselves are tar like, but depending on where the rotate kicks off they may not be extractable via `tar` without the leading archive